src/the-2021-scheduler.md

```diff
@@ -32,7 +32,7 @@ day, and may show them in a different order.
 
 The v3 scheduler uses Anki's new undo infrastructure: you can answer a card,
 bury another card, answer a different card, then undo each action in turn. Previous schedulers handled undo for review and non-review actions separately, so a non-review action
-would clear the review history, and vice versa.
+would clear the review history, or the other way around.
```

Comment:
I don't think "or the other way around" makes sense here. It sounds like you're saying, "the review history would clear a non-review action." "Vice versa" is better, but if you don't want to keep that -- 

- make it a "so for example" and drop that phrase from the end. 
- make it generic -- "so performing an action for one would clear the undo history for the other."